client/core/core.go

```diff
@@ -6616,6 +6670,33 @@ func (c *Core) handleReconnect(host string) {
 		// Continue to resubscribe to market fees.
 	}
 
+	// Now that reconcileTrades has been run in authDEX, make a list of epoch
+	// status orders that should be re-checked in the next epoch because we may
+	// have missed the preimage request while disconnected.
+	epochOrders := make(map[string][]*trackedTrade)
+	for _, trade := range dc.trackedTrades() {
+		if trade.status() == order.OrderStatusEpoch {
+			epochOrders[trade.mktID] = append(epochOrders[trade.mktID], trade)
+		}
+	}
+	for mkt := range epochOrders {
+		trades := epochOrders[mkt] // don't capture loop var below
+		time.AfterFunc(
```

Comment:
`tick`s are a little too infrequent for my taste.

This should be fine even if there are frequent reconnects because the `func` is mostly a no-op if the orders are no longer epoch status.

It's also appealing to grab a local slice of the orders rather than having to add more fields to `dexConnection` and then iterate through `dc.trades` in `tick`.  I'm open to exploring a specific suggestion for an approach that uses `tick` however.